Index: app/src/main/java/com/aula/exameperiodico/AreaRestrita.java
===================================================================
diff --git a/app/src/main/java/com/aula/exameperiodico/AreaRestrita.java b/app/src/main/java/com/aula/exameperiodico/ui/arearestrita/AreaRestrita.java
rename from app/src/main/java/com/aula/exameperiodico/AreaRestrita.java
rename to app/src/main/java/com/aula/exameperiodico/ui/arearestrita/AreaRestrita.java
--- a/app/src/main/java/com/aula/exameperiodico/AreaRestrita.java	(revision 5c29df58db10b9a10a766577f9b45762cee4bf9d)
+++ b/app/src/main/java/com/aula/exameperiodico/ui/arearestrita/AreaRestrita.java	(date 1748964150963)
@@ -1,4 +1,4 @@
-package com.aula.exameperiodico;
+package com.aula.exameperiodico.ui.arearestrita;
 
 import android.os.Bundle;
 
@@ -8,6 +8,8 @@
 import android.view.View;
 import android.view.ViewGroup;
 
+import com.aula.exameperiodico.R;
+
 /**
  * A simple {@link Fragment} subclass.
  * Use the {@link AreaRestrita#newInstance} factory method to
